Source/WebKit/Shared/ApplePay/WebPaymentCoordinatorProxy.h

```diff
@@ -203,6 +204,10 @@ class WebPaymentCoordinatorProxy
         CouponCodeChanged,
 #endif
 
+        // Inactivating - Could not complete the payment and is about to idle.
+        // Currently only transitions here when the web process terminates while the payment coordinator is active.
+        Inactivating,
```

Comment:
Nit - maybe one of `Deactivating`, or `BecomingInactive`?

Reply:
English is hard!

Reply:
Fixed in 74a5c4c